Misc/NEWS.d/next/Library/2019-03-16-17-59-18.bpo-31062.8vxQyS.rst

```diff
@@ -0,0 +1,2 @@
+Allow the `socket.makefile` method to handle line buffering properly
+(`buffering=1`). This is done by mimicking the behavior of `io.open`.
```

Comment:
```rst
:func:`open`
```
Also, the markup for `buffering=1` should use double backquotes (https://devguide.python.org/documenting/#inline-markup).

Reply:
Fixed in 030c991.